src/components/Login/index.tsx

```diff
@@ -55,6 +55,7 @@ const LoginModalContent = (props: Props) => {
       case 2:
```

Comment:
Do you think using an object or enum instead of step 1-6 is a better idea and easier to debug later?

Reply:
As far as I read. There are several components using these steps by passed props. Using these numbers makes debugging harder than an object or enum.

Reply:
![image](https://user-images.githubusercontent.com/112835611/194704726-4cabe6d7-00a3-450e-9268-9ff0b65f0081.png)
I have used enum for stepName. I think it is clearer than just using a number.